core/email_api.php

```diff
@@ -308,6 +308,20 @@ function email_collect_recipients( $p_bug_id, $p_notify_type, array $p_extra_use
 		}
 	}
 
+	# add Category Owner
+	if( ON == email_notify_flag( $p_notify_type, 'category' ) ) {
+		$t_category_id = bug_get_field( $p_bug_id, 'category_id' );
+
+		if( $t_category_id > 0 ) {
+			$t_category_assigned_to = category_get_field( $t_category_id, 'user_id' );
+
+			if( $t_category_assigned_to > 0 ) {
```

Comment:
Should we check that category owner has access to the issue? e.g. what if the issue is private, but category owner only has access to public issues?  What if the category is global and this owner doesn't have access to the project containing the issue?